src/components/templates/ResourceTemplateRow.jsx

```diff
@@ -20,16 +21,7 @@ const ResourceTemplateRow = ({ row, handleClick, handleCopy, handleEdit }) => {
   return (
     <tr key={row.id}>
       <td style={{ wordBreak: "break-all" }} data-testid="name">
-        {canCreate ? (
-          <Link
-            to={{ pathname: "/editor", state: {} }}
-            onClick={(e) => handleClick(row.id, e)}
-          >
-            {row.resourceLabel}
-          </Link>
-        ) : (
-          <span>{row.resourceLabel}</span>
-        )}
+        <span>{row.resourceLabel}</span>
```

Comment:
Unlink the label in the first column (this used to be the way to create a resource)